src/index.ts

```diff
@@ -231,6 +231,13 @@ https://blog.newrelic.com/product-news/aws-lambda-extensions-integrations/
         `Serverless ${version} does not support layers. Please upgrade to >=1.34.0.`
       );
       return;
+    } else if (semver.lt(version, "3.0.0")) {
+      this.log.error(
+        `Serverless ${version} and Serverless >= 3.0.0 have some incompatibilities in logging.  
+        This plugin is optimized for Serverless 3.x. Please upgrade Serverless to >=3.0.0, 
```

Comment:
`optimized` isn't the same as `requires`. Let's be clear here.

Also, just checking: does `this.log.error` work with 2.x?